R/pkg/R/mllib.R

```diff
@@ -1233,7 +1233,7 @@ setMethod("spark.survreg", signature(data = "SparkDataFrame", formula = "formula
 #'        docConcentration. Only 1-size or \code{k}-size numeric is accepted.
 #' @param customizedStopWords stopwords that need to be removed from the given corpus. Ignore the
 #'        parameter if libSVM-format column is used as the features column.
-#' @param maxVocabSize maximum vocabulary size, default 1 << 18
+#' @param maxVocabSize maximum vocabulary size, default is 1 << 18
```

Comment:
`maxVocabSize maximum vocabulary size, default 1 << 18` could still be useful since the value in the signature is `bitwShiftL(1, 18)`